### ISSUE TITLE
Broken anchors - Understanding Conformance

### ISSUE THREAD

**Issue by luce-carevic** [w3c/wcag#3064]
Hi!

There are broken anchors on the [Understanding Conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance) new page. 

Scope : 

- all the anchors "#dfn-[term]" for example "#dfn-web-page";
- anchor on the word "technologies" (section Understanding Requirement 5)

**Comment by alastc** [comment]
@michael-n-cooper I assume this is a generator issue (the definitions in the content don't work). 

Either the definitions need adding to the page, or they should go to the versions in the index page.

**Comment by EricDunsworth** [comment]
There are also various placeholder links (e.g. ``<a>link</a>``) throughout that page.

For example, the [understanding conforming alternate versions section](https://www.w3.org/WAI/WCAG21/Understanding/conformance#conforming-alt-versions)'s "functionality" and "human languages" links have CSS hover effects, but can't be activated.

The equivalent links in [WCAG 2.0's version of that section](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-conforming-alt-versions-head) point to the [glossary section in WCAG 2.0's main page](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#glossary). Their anchor destinations are formatted as ``#[term]def`` .

**Comment by fstrr** [comment]
This issue is now on the 2.1 and 2.2 versions of the site.

Links in the "Page Contents" sidebar work, but inline links starting with `#dfn-` don't.

I've checked the [2.0 version of this page](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html) to find out where links should be going to and think this is likely a templating issue.

**Comment by patrickhlauke** [comment]
Indeed it's a template issue ... in essence, when normative SC wording contains links to definitions, they work fine when in the spec (since the spec itself has all the definitions included at the end), but when the SC fragment itself is pulled into the understanding documents, the definitions aren't always pulled into the final understanding page that's being generated. /cc @iadawn

**Comment by fstrr** [comment]
There are some `id` attributes in the page's HTML that can be updated to make some of the links work, but that doesn't fix the whole page, and I didn't want to put in a "half a job" PR.

**Comment by kfranqueiro** [comment]
This was resolved for 2.2 by #3917; it will be resolved for 2.1 when we can re-generate docs for that version with the new build system.

**Comment by kfranqueiro** [comment]
This is resolved for 2.1 as well now that WCAG 2.1 informative docs have been republished via the new build system, made possible via https://github.com/w3c/wcag/pull/4007.
